alf/algorithms/data_transformer.py

```diff
@@ -625,6 +625,27 @@ def forward(self, reward):
         return reward.clamp(*self._minmax)
 
 
+@alf.configurable
+class RewardShifting(RewardTransformer):
+    """Shift immediate rewards by a displacement of ``bias``.
+
+    Can be used as a reward shaping function passed to an algorithm
+    (e.g. ``ActorCriticAlgorithm``).
+    """
+
+    def __init__(self, observation_spec=(), bias=0.0):
```

Comment:
If someone is intended to use it, I would assume they'd always want to provide a `bias`. Might be good to make it a mandatory argument?

Reply:
Good point. Now change to be a mandatory argument. Also did the same change to ``RewardScaling``.

Reply:
Pushed another commit edf24bacbe16ae7c1fdd02604e34cd8248fa6475 to fix an issue caused by the mandatory argument change.